data/assets/scene/solarsystem/planets/earth/lagrange_points/lagrange_points.asset

```diff
@@ -2,3 +2,13 @@ asset.require("scene/solarsystem/planets/earth/lagrange_points/L1")
 asset.require("scene/solarsystem/planets/earth/lagrange_points/L2")
 asset.require("scene/solarsystem/planets/earth/lagrange_points/L4")
 asset.require("scene/solarsystem/planets/earth/lagrange_points/L5")
+
+
+asset.meta = {
+  Name = "Earth Lagrange Points",
+  Version = "1.0",
+  Description = [[ Meta asset for the earth lagrange_points. Contains L1,L2,L4 and L5]],
```

Comment:
"Meta asset to include all the Earth Lagrange points. Includes L1, L2, L4 and L5."

Less "code-y" :)